packages/db/src/translate/providers.ts

```diff
@@ -41,6 +41,9 @@ export const translateApiProviderToPostgresProvider = async (
     port: record.port,
     host: record.host,
     protocol: record.protocol,
+    allowed_redirects: record.allowedRedirects
```

Comment:
It shouldn't block this PR, but if providers is in the feature branch, we should either make a point to sort it in the feature under 2548 *or* write a ticket to do so for the phase 2 work.

Reply:
What do we need to sort on the feature branch?

Reply:
Sorry! 😅 the postgres->api translation to make sure we're not dropping the value/returning the wrong key.

Reply:
Ah okay. Yeah I can do a follow up PR